duel.py

```diff
@@ -49,7 +49,7 @@ def duel(bot, channel, instigator, target, is_admin=False, warn_nonexistent=True
         return module.NOLIMIT
     target_unduelable = get_unduelable(bot, target)
     if target_unduelable and not is_admin:
-        bot.say("You SHALL NOT duel %s!" % target)
+        bot.say("You SHALL NOT duel this person!")
```

Comment:
`s/this/that/` at least. Might even make sense to say `user` instead of `person`, hmm…